ts/store/reducers/wallet/wallets.ts

```diff
@@ -215,6 +215,20 @@ export const getPagoPAMethodsSelector = createSelector(
     )
 );
 
+// return those payment methods that have BPD as enabled function
+export const getBpdAMethodsSelector = createSelector(
```

Comment:
```suggestion
export const getBpdMethodsSelector = createSelector(
```
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏Maybe a typo?

Reply:
Good catch! Thank you, fixed in [4b58ef7](https://github.com/pagopa/io-app/pull/3725/commits/4b58ef73f17e9f3017f1ddbe2cbb92efd9106bc7)